src/System.Text.Http/System/Text/Http/Parser/HttpParser.cs

```diff
@@ -27,7 +24,7 @@ public class HttpParser : IHttpParser
         private const byte BytePercentage = (byte)'%';
         private const long maxRequestLineLength = 1024;
 
-        static readonly byte[] s_Eol = Encoding.UTF8.GetBytes("\r\n");
+        private static ReadOnlySpan<byte> Eol => new byte[] { ByteCR, ByteLF };
```

Comment:
We  could probably make the same change to the other static byte arrays below (if it makes sense): `s_http11`/etc.

Reply:
I don't think we're using them, but I'll look.